src/pybind/mgr/influx/module.py

```diff
@@ -178,12 +178,12 @@ def get_df_stats(self, now):
         pool_info = {}
 
         df_types = [
-            'bytes_used',
+            'stored',
             'kb_used',
             'dirty',
             'rd',
             'rd_bytes',
-            'raw_bytes_used',
+            'stored_raw',
```

Comment:
and https://github.com/ceph/ceph/blob/590babe82645ca901d69f2a166cb55000890ef8e/qa/tasks/mgr/dashboard/test_health.py#L102